services/manifest-repo-export-service/pkg/repository/transformer.go

```diff
@@ -1458,13 +1458,13 @@ func getEnvironmentGroupsEnvironmentsOrEnvironment(configs map[string]config.Env
 			envGroupConfigs[env] = config
 		}
 	}
-	if len(envGroupConfigs) == 0 {
+	if isTargetEnvGroup || len(envGroupConfigs) == 0 {
```

Comment:
I'm not sure if this logic is correct.
I would prefer something like this on top level of this function:
if isTargetEnvGroup {
  /// look for group
} else {
 // look for env
}